### PR TITLE
Add typing for modeler events

### LINKED ISSUE #866: Add interface for the modeler events
#### Describe your issue

The `eventBus` of the `modeler` components offers many events where we can subscribe to. We should define them all in the `constants.ts` which can be found in the `bpmnmodeler` folder of the contracts, rather then just accessing them with _magic strings_. 

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

### PR DESCRIPTION
**Pull request by LeoTT**
:tada: **Thanks for submitting this Pull Request**

Please provide a list including your changes.

To help us to create a meaningful Changelog, we agreed 
to use an emoji-based commit-style. Use the [Emojis](#emojis)
we provided below.

Fill out the following, provide reference to issues
and this PR- we will use this as merge commit message-body
and the title of this PR as commit-headline:

---

**Changes:**

1. Adds list of modeler events used by the eventbus of camunda modeler.

I can not guarantee that all possible eventnames are included. At some points the names are dynamically generated.

I generated this list with code search tools and added eventnames that @Vyperus found by logging.

**Issues:**

Closes #866 

PR: #925

---

See the [examples](#example) for inspiration.

## How can others test the changes?

This adds no runtime behaviour. You could test `au run build`. If I broke anything you would see it there.

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [ ] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

## Example

<details>
<summary>
:warning: Before merging please click here to expand and follow the guide.
</summary>
<br>

Please use `:twisted_rightwards_arrows:` at the beginning of your merge commit title.

Example 1:

<pre>
<code>
:twisted_rightwards_arrows: :bug: Fix Wrong Text Decoration at ...
</code>
</pre>

To get your commit message, just copy the first part of this pull request.

Example 2:
<pre>
<code>
**Changes:**

- Fixes Wrong Text Decoration at ...
- Fixes some typos
- ...

**Issues:**

Closes #NumberOfFixedIssue

PR: #NumberOfThisPR
</code>
</pre>
</details>

## Emojis

<details>
<summary>
Expand for a list of most used Emojis.
</summary>
<br>

Please prefix your commit messages with an Emoji.

Ref: https://gitmoji.carloscuesta.me/

| Description              | Glyphe               | Emoji  |
|--------------------------|----------------------|--------|
| Bugfix                   | `:bug:`              | 🐛     |
| Configuration releated   | `:wrench:`           | 🔧     |
| Cosmetic                 | `:lipstick:`         | 💄     |
| Dependencies Downgrade   | `:arrow_down:`       | ⬇️     |
| Dependencies Upgrade     | `:arrow_up:`         | ⬆️     |
| Formatting               | `:art:`              | 🎨     |
| Improving Performance    | `:zap:`              | ⚡️     |
| Initial commit           | `:tada:`             | 🎉     |
| Linter                   | `:rotating_light:`   | 🚨     |
| Miscellaneous            | `:package:`          | 📦     |
| New Feature              | `:sparkles:`         | ✨     |
| Refactoring Code         | `:recycle:`          | ♻️     |
| Releasing / Version tags | `:bookmark:`         | 🔖     |
| Removing Stuff           | `:fire:`             | 🔥     |
| Tests                    | `:white_check_mark:` | ✅     |
| Work In Progress (WIP)   | `:construction:`     | 🚧     |

</details>

### CODE REVIEW

File: src/contracts/eventbus/IEventBus.ts [hunk 1]
```diff
@@ -1,21 +1,139 @@
+export type EventName = 'attach'
```
Comment (Paulomart) [inline src/contracts/eventbus/IEventBus.ts]: Could you add a source where they are comming from? Maybe we need to update them in the future.

**Comment by Vyperus** [comment]
[bpmn-js eventBus events.zip](https://github.com/process-engine/bpmn-studio/files/2371161/bpmn-js.eventBus.events.zip)

This should be all.

File: src/contracts/eventbus/IEventBus.ts (hunk 1 above)
Comment (LeoTT) [reply src/contracts/eventbus/IEventBus.ts]: Could not find a source. Crawled through the codebase with silver searcher and used some editor macros to sort them, format them and delete duplicates.

Comment (Paulomart) [reply src/contracts/eventbus/IEventBus.ts]: Maybe you could still add a comment, how to find them?

File: src/contracts/eventbus/IEventBus.ts [hunk 2]
```diff
@@ -1,21 +1,252 @@
...
+  once(events: Array<EventNameType> | EventNameType,
        callback: Function,
        callbackScope: any): void;
-  off(event: string,
+  off(event: EventNameType,
       callback?: Function): void;
-  fire(nameOrType: string | {type: string},
+  fire(nameOrType: EventNameType | {type: EventNameType},
        data?: any): any;
 }
```
Comment (cmg-dev) [inline src/contracts/eventbus/IEventBus.ts]: Wouldn't this be a nice contribution to the typings repo?

AKA the open source contribution to BPMN.io and the typescript typings repo?!

File: src/contracts/eventbus/IEventBus.ts (hunk 1 above)
Comment (LeoTT) [reply src/contracts/eventbus/IEventBus.ts]: Got many event names from @Vyperus, who got them by logging. I don't have a clean recipe for that.

**Pull request merged**